tests_and_analysis/test/euphonic_test/test_force_constants_calculate_qpt_phonons_phonopy.py

```diff
@@ -0,0 +1,138 @@
+from euphonic import ForceConstants
+import pytest
+import os
+from ..utils import get_data_path
+import numpy.testing as npt
+import numpy as np
+from euphonic import ureg
+
+
+@pytest.mark.integration
+class TestCalculateQPointPhononModes:
+
+    phonopy_yaml_file = os.path.join(
+        get_data_path(), "phonopy_data", "NaCl", "interpolation"
+    )
```

Comment:
For testing our functions we should be reading from a .json file, like in `test_crystal.py`

Reply:
Refactored to use json